PostHog/Replay/PostHogSessionReplayConfig.swift

```diff
@@ -42,7 +42,19 @@
         /// This is used for capturing the view as a wireframe or screenshot
         /// The lower the number more snapshots will be captured but higher the performance impact
         /// Defaults to 1s
-        @objc public var debouncerDelay: TimeInterval = 1.0
+        @available(*, deprecated, message: "Deprecated in favor of 'throttleDelay' which provides identical functionality. Will be removed in the next major release.")
```

Comment:
https://posthog.com/docs/session-replay/installation#configuration-options
lets fix the docs as well